pkg/cloud/azure/actuators/cluster/actuator_test.go

```diff
@@ -157,27 +157,35 @@ func TestServicesCreatedCount(t *testing.T) {
 	if cache[azure.GenerateVnetName(fakeReconciler.scope.Cluster.Name)] != 1 {
 		t.Errorf("Expected 1 count of %s service", azure.GenerateVnetName(fakeReconciler.scope.Cluster.Name))
 	}
+
```

Comment:
what are all these pluses?

Reply:
just adding spaces between the `if` statements.